python/ui/setup_project.py

```diff
@@ -612,7 +612,7 @@ def retranslateUi(self, Wizard):
         self.additional_details_button.setText(QtGui.QApplication.translate("Wizard", "Show Details", None, QtGui.QApplication.UnicodeUTF8))
         self.final_message.setText(QtGui.QApplication.translate("Wizard", "Toolkit is now ready to use in your Project!<br/>\n"
 "<br/>\n"
-"For more information on how to configure your Project and tailor it to your system, <a href=\"https://support.shotgunsoftware.com/entries/95443887#Welcome%20to%20Toolkit\" style=\"color: rgb(203, 205, 205);\">click here to check out our documentation</a>.", None, QtGui.QApplication.UnicodeUTF8))
+"For more information on how to configure your Project and tailor it to your system, <a href=\"https://developer.shotgridsoftware.com/1b9c259a/?title=Launch+App#Welcome%20to%20Toolkit\" style=\"color: rgb(203, 205, 205);\">click here to check out our documentation</a>.", None, QtGui.QApplication.UnicodeUTF8))
```

Comment:
This is about tk-framework-adminui (advanced project setup dialog) I think the right URL should be https://developer.shotgridsoftware.com/5d83a936/?title=Configuration+Setup